docs/reference/ml/anomaly-detection/apis/get-category.asciidoc

```diff
@@ -48,7 +48,12 @@ include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=job-id-anomaly-detection]
 
 `<category_id>`::
 (Optional, long) Identifier for the category. If you do not specify this
-parameter, the API returns information about all categories in the {anomaly-job}.
+parameter, the API returns information about all categories in the {anomaly-job},
+or all categories in the {anomaly-job} for the specified partition if
+`partition_field_value` is supplied. Since `category_id` is unique at the job
```

Comment:
> Since `category_id` is unique at the job...

IMO, this sentence is not necessary if you just change first phrase like this:

> Identifier for the category, which is unique in the job.